source/memory/memory.go

```diff
@@ -19,17 +19,35 @@ package memory
 import (
 	"fmt"
 	"io/ioutil"
+	"log"
+	"os"
 	"strings"
 
 	"sigs.k8s.io/node-feature-discovery/source"
 )
 
+var logger = log.New(os.Stderr, "", log.LstdFlags)
```

Comment:
No need to initialize a logger. You can just use log.Printf() which utilizes the default logger. I did a similar change for the other sources in #150

Reply:
Done.